cmd/keymasterd/app.go

```diff
@@ -445,6 +445,14 @@ func getUserFromRequest(r *http.Request) string {
 	return user
 }
 
+func (ai *authInfo) expires() string {
+	if ai.ExpiresAt.IsZero() {
+		return ""
+	}
+	return fmt.Sprintf("at: %s (in: %s)",
+		ai.ExpiresAt, time.Until(ai.ExpiresAt).Round(time.Second).String())
```

Comment:
what time zone?... why not send data as epoch (int) and let JS tranform it to the right display zone? For people not using JS this will make sense for using using JS it will be more familiar... also this is on the admin path.. need to read more before approval

Reply:
It's the timezone Keymaster is running in.
It's on the admin path simply because we want to continue showing the session expiration time on the admin pages too.